docs/diagrams/ModuleClassModelClassDiagram.puml

```diff
@@ -0,0 +1,25 @@
+@startuml
+!include style.puml
+show members
+hide empty members
+skinparam classAttributeFontColor #FFFFFF
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+Class ModuleClass
+
+package ModuleClass <<Rectangle>> {
+    class ModuleClass {
+        studentUuids : Set<UUID>
+    }
+    class Name {
+        fullName : String
+    }
```

Comment:
Is there a particular reason why this differs from the Student diagram?

Also, I think it is not part of the `ModuleClass` package?